Tools/c-analyzer/cpython/_parser.py

```diff
@@ -47,255 +41,240 @@ def clean_lines(text):
+    ('Include/cpython/ceval.h', 'Py_CPYTHON_CEVAL_H', '1'),
+    ('Include/cpython/code.h', 'Py_CPYTHON_CODE_H', '1'),
+    ('Include/cpython/dictobject.h', 'Py_CPYTHON_DICTOBJECT_H', '1'),
+    ('Include/cpython/fileobject.h', 'Py_CPYTHON_FILEOBJECT_H', '1'),
+    ('Include/cpython/fileutils.h', 'Py_CPYTHON_FILEUTILS_H', '1'),
+    ('Include/cpython/frameobject.h', 'Py_CPYTHON_FRAMEOBJECT_H', '1'),
+    ('Include/cpython/import.h', 'Py_CPYTHON_IMPORT_H', '1'),
+    ('Include/cpython/interpreteridobject.h', 'Py_CPYTHON_INTERPRETERIDOBJECT_H', '1'),
+    ('Include/cpython/listobject.h', 'Py_CPYTHON_LISTOBJECT_H', '1'),
+    ('Include/cpython/methodobject.h', 'Py_CPYTHON_METHODOBJECT_H', '1'),
+    ('Include/cpython/object.h', 'Py_CPYTHON_OBJECT_H', '1'),
+    ('Include/cpython/objimpl.h', 'Py_CPYTHON_OBJIMPL_H', '1'),
+    ('Include/cpython/pyerrors.h', 'Py_CPYTHON_ERRORS_H', '1'),
+    ('Include/cpython/pylifecycle.h', 'Py_CPYTHON_PYLIFECYCLE_H', '1'),
+    ('Include/cpython/pymem.h', 'Py_CPYTHON_PYMEM_H', '1'),
+    ('Include/cpython/pystate.h', 'Py_CPYTHON_PYSTATE_H', '1'),
+    ('Include/cpython/sysmodule.h', 'Py_CPYTHON_SYSMODULE_H', '1'),
+    ('Include/cpython/traceback.h', 'Py_CPYTHON_TRACEBACK_H', '1'),
+    ('Include/cpython/tupleobject.h', 'Py_CPYTHON_TUPLEOBJECT_H', '1'),
+    ('Include/cpython/unicodeobject.h', 'Py_CPYTHON_UNICODEOBJECT_H', '1'),
+
+    # implied include of <unistd.h>
+    ('Include/**/*.h', '_POSIX_THREADS', '1'),
+    ('Include/**/*.h', 'HAVE_PTHREAD_H', '1'),
+
+    # from pyconfig.h
+    ('Include/cpython/pthread_stubs.h', 'HAVE_PTHREAD_STUBS', '1'),
+    ('Python/thread_pthread_stubs.h', 'HAVE_PTHREAD_STUBS', '1'),
+
+    # from Objects/bytesobject.c
+    ('Objects/stringlib/partition.h', 'STRINGLIB_GET_EMPTY()', 'bytes_get_empty()'),
+    ('Objects/stringlib/join.h', 'STRINGLIB_MUTABLE', '0'),
+    ('Objects/stringlib/partition.h', 'STRINGLIB_MUTABLE', '0'),
+    ('Objects/stringlib/split.h', 'STRINGLIB_MUTABLE', '0'),
+    ('Objects/stringlib/transmogrify.h', 'STRINGLIB_MUTABLE', '0'),
+
+    # from Makefile
+    ('Modules/getpath.c', 'PYTHONPATH', '1'),
+    ('Modules/getpath.c', 'PREFIX', '...'),
+    ('Modules/getpath.c', 'EXEC_PREFIX', '...'),
+    ('Modules/getpath.c', 'VERSION', '...'),
+    ('Modules/getpath.c', 'VPATH', '...'),
+    ('Modules/getpath.c', 'PLATLIBDIR', '...'),
+    # ('Modules/_dbmmodule.c', 'USE_GDBM_COMPAT', '1'),
+    ('Modules/_dbmmodule.c', 'USE_NDBM', '1'),
+    # ('Modules/_dbmmodule.c', 'USE_BERKDB', '1'),
+
+    # See: setup.py
+    ('Modules/_decimal/**/*.c', 'CONFIG_64', '1'),
+    ('Modules/_decimal/**/*.c', 'ASM', '1'),
+    ('Modules/expat/xmlparse.c', 'HAVE_EXPAT_CONFIG_H', '1'),
+    ('Modules/expat/xmlparse.c', 'XML_POOR_ENTROPY', '1'),
+    ('Modules/_dbmmodule.c', 'HAVE_GDBM_DASH_NDBM_H', '1'),
+
+    # others
+    ('Modules/_sre/sre_lib.h', 'LOCAL(type)', 'static inline type'),
+    ('Modules/_sre/sre_lib.h', 'SRE(F)', 'sre_ucs2_'),  ##F
```

Comment:
This produces identical output to the current state, due to parsing `##F` as a comment, but I think the line should probably read:

```suggestion
    ('Modules/_sre/sre_lib.h', 'SRE(F)', 'sre_ucs2_##F'),
```

Reply:
good catch!